src/main/java/cc/oakk/lotto/model/LottoResults.java

```diff
@@ -0,0 +1,55 @@
+package cc.oakk.lotto.model;
+
+import cc.oakk.lotto.view.printer.Printable;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static cc.oakk.lotto.util.ValidationAdapters.throwIfNull;
+
+public class LottoResults implements Printable<LottoResults> {
+    private final Map<Rank, List<LottoResult>> rankMap;
```

Comment:
인스턴스 변수라도, 명확한 구현체의 이름을 사용하는건 지양해주세요 :)
```suggestion
    private final Map<Rank, List<LottoResult>> ranks;
```